BlueChip/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "BlueChip",
```

Comment:
Please remove all capital letters from directory names and `chain_name`s (BlueChip -> bluechip and Images -> images)